client/src/components/Workbench/Workbench.scss

```
@@ -47,6 +47,17 @@
 .workbench-content .wb-link {
   color: var(--global-color-accent--normal);
 }
+.workbench-content .wb-link:has(.icon):hover {
+  text-decoration: unset;
+}
+
+.workbench-content .wb-link:has(.icon) > span {
+  text-decoration: unset;
+}
+
+.workbench-content .wb-link:has(.icon):hover > span {
+  text-decoration: underline;
+}
```

Suggestions:
1. Combine similar-purpose selectors with identical rulesets.
2. Remove new lines between similar-purpose selectors.
    - To visually group them, especially in a file that has no organization. 

This code change may conflict with my other suggestions, so consider manually making any related change.

```suggestion
.workbench-content .wb-link:has(.icon):hover,
.workbench-content .wb-link:has(.icon) > span {
  text-decoration: unset;
}
.workbench-content .wb-link:has(.icon):hover > span {
  text-decoration: underline;
}
```